cpp/src/Ice/Exception.cpp

```diff
@@ -539,6 +466,59 @@ Ice::Exception::ice_stackTrace() const
     return getStackTrace(*_stackFrames);
 }
 
+bool
+Ice::Exception::ice_enableStackTraceCollection()
```

Comment:
Is the `ice_` prefix really necessary?

Reply:
We use this ice_ prefix for all other member functions. No ice_ prefix would be inconsistent.